services/httpd/handler.go

```diff
@@ -218,6 +218,10 @@ func NewHandler(c Config) *Handler {
 			"status-head",
 			"HEAD", "/status", false, true, authWrapper(h.serveStatus),
 		},
+		Route{ // Ping
+			"ping",
+			"GET", "/health", false, true, authWrapper(h.serveHealth),
```

Comment:
Is this under `/api/v2/health` in 2.0? If so, should it be the same here? E.g., we added `/api/v2/write` for 1.8.

Reply:
https://v2.docs.influxdata.com/v2.0/api/#tag/Health

I believe the feedback is that we need a /health API in order for the 2.0 clients to successfully perform writes?

Reply:
The `/health` endpoint is mapped to root path in 2.0. See: https://github.com/influxdata/influxdb/blob/c8aabaf515dccea989c921ac3d2140beede9fbb9/http/handler.go#L22

We need this because a lot of users uses `/ping` and `/health` endpoint to check if is InfluxDB running... and after success they start writing or querying.